### PR TITLE
None

### PR DESCRIPTION
**Pull request by pbrunet**

### CODE REVIEW

File: pythran/spec.py [hunk 1]
```diff
@@ -221,6 +224,8 @@ def p_term(self, p):
         from numpy import uint8, uint16, uint32, uint64
 
         p[0] = eval(p[1])
+        if p[0] is None:
+            p[0] = type(None)
```
Comment (serge-sans-paille) [inline pythran/spec.py]: you should update the doc about pythran annotation too

File: pythran/pythonic/__builtin__/dict/get.hpp [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #include "pythonic/include/__builtin__/dict/get.hpp"
 
 #include "pythonic/types/dict.hpp"
-#include "pythonic/types/none.hpp"
+#include "pythonic/types/NoneType.hpp"
```
Comment (serge-sans-paille) [inline pythran/pythonic/__builtin__/dict/get.hpp]: Why this change?

Comment (pbrunet) [inline pythran/pythonic/__builtin__/dict/get.hpp]: Because Pythran generate includes with ```pythonic/types/type(input_type).h```
Here , input_type for None is NoneType. I prefer have the same naming convention for every type than a special handling for None.

Comment (serge-sans-paille) [inline pythran/pythonic/__builtin__/dict/get.hpp]: ok great!
